api/spec/v1.yaml

```
@@ -2572,6 +2596,47 @@ components:
             the `/{runtime}/accounts/{address}` endpoint.
           example: false
 
+    EvmNftList:
+      allOf:
+        - $ref: '#/components/schemas/List'
+        - type: object
+          required: [evm_nfts]
+          properties:
+            evm_nfts:
+              type: array
+              items:
+                $ref: '#/components/schemas/EvmNft'
+              description: A list of L2 EVM NFT (ERC-721, ...) instances.
+          description: A list of NFT instances.
+
+    EvmNft:
+      type: object
+      required:
+        - token
+        - contract_addr
+        - id
+      properties:
+        token:
+          $ref: '#/components/schemas/EvmToken'
+        id:
+          <<: *BigIntType
```

```suggestion
          <<: *BigIntType
          description: The instance ID of this NFT within the collection represented by `token`.
```

WDYT about this as an attempt to tie together the terminology / make our choices a little more explicit?

---

sounds good, thanks